### PR TITLE
ENH: improve boilerplate content

### PR DESCRIPTION
**Pull request by TheColorDarkBlue**

### CODE REVIEW

File: rendercv/data_models.py [hunk 1]
```diff
@@ -903,11 +903,12 @@ def url(self) -> str:
                 "GitHub": "https://github.com/",
                 "GitLab": "https://gitlab.com/",
                 "Instagram": "https://instagram.com/",
-                "Orcid": "https://orcid.org/",
+                "ORCID": "https://orcid.org/",
                 "Twitter": "https://twitter.com/",
                 "StackOverflow": "https://stackoverflow.com/users/",
                 "ResearchGate": "https://researchgate.net/profile/",
                 "YouTube": "https://youtube.com/",
+                "Google Scholar": "https://scholar.google.com/citations?user=",
```
Comment (TheColorDarkBlue) [inline rendercv/data_models.py:912]: Not exactly sure where else to update in order to pass tests

File: tests/test_data_models.py [hunk 2]
```diff
@@ -427,7 +427,7 @@ def test_invalid_social_networks(network, username):
         ("LinkedIn", "myusername", "https://linkedin.com/in/myusername"),
         ("GitHub", "myusername", "https://github.com/myusername"),
         ("Instagram", "myusername", "https://instagram.com/myusername"),
-        ("Orcid", "myusername", "https://orcid.org/myusername"),
```
Comment (TheColorDarkBlue) [inline tests/test_data_models.py:430]: Not exactly sure where else to update in order to pass tests. To clarify the change was properly punctuating ORCID

File: rendercv/data_models.py (hunk 1 above)
Comment (sinaatalay) [inline rendercv/data_models.py:912]: Adding a new social network requires other changes as well. I will document this in the developer guide; for now, we can omit this line for this pull request.

File: rendercv/data_models.py [hunk 3]
```diff
@@ -1589,65 +1583,65 @@ def get_a_sample_data_model(
         "projects": [
             NormalEntry(
                 name="Multi-User Drawing Tool",
-                date="2004",
+                date="2004",  # I would omit start/end dates and replace with a project link as the date is largely irrelevant
```
Comment (sinaatalay) [inline rendercv/data_models.py:1586]: Maybe to showcase that RenderCV's date field doesn't need to be dates, we can do this:

date="[github.com/user/example](https://github.com/user/example)"

Also, we shouldn't leave any discussions as comments in the source code.

File: rendercv/data_models.py [hunk 4]
```diff
@@ -1589,65 +1583,65 @@ def get_a_sample_data_model(
...
+                # project_link = "github.com/username/repo"
```
Comment (sinaatalay) [inline rendercv/data_models.py:1587]: We shouldn't include this line.

File: rendercv/data_models.py [hunk 5]
```diff
@@ -1589,65 +1583,65 @@ def get_a_sample_data_model(
...
                     ),
-                    "Used C++ and MFC.",
+                    "Tools Used: C++, MFC",
                 ],
             ),
             NormalEntry(
                 name="Synchronized Calendar",
                 start_date="2003",
                 end_date="2004",
+                # project_link = "github.com/username/repo",
```
Comment (sinaatalay) [inline rendercv/data_models.py:1601]: We shouldn't include this line.

File: rendercv/data_models.py [hunk 6]
```diff
@@ -1589,65 +1583,65 @@ def get_a_sample_data_model(
...
                     ),
-                    "Used C#.NET, SQL, and XML.",
+                    "Tools Used: C#, .NET, SQL, XML",
                 ],
             ),
             NormalEntry(
                 name="Operating System",
-                date="2002",
+                date="2002",  # would replace project dates with a project link as the date is largely irrelevant
+                # project_link = "github.com/username/repo",
```
Comment (sinaatalay) [inline rendercv/data_models.py:1614]: We shouldn't include this line.

File: rendercv/data_models.py [hunk 7]
```diff
@@ -1589,65 +1583,65 @@ def get_a_sample_data_model(
...
+                        " with other users"
                     ),
-                    "Used C#.NET, SQL, and XML.",
+                    "Tools Used: C#, .NET, SQL, XML",
                 ],
             ),
             NormalEntry(
                 name="Operating System",
-                date="2002",
+                date="2002",  # would replace project dates with a project link as the date is largely irrelevant
```
Comment (sinaatalay) [inline rendercv/data_models.py:1613]: See the comment above.

**Comment by sinaatalay** [comment]
Thank you very much for the help! I've added some comments and will then proceed with merging this.

File: rendercv/data_models.py (hunk 1 above)
Comment (TheColorDarkBlue) [reply rendercv/data_models.py:912]: Ok so should i make a new commit removing this new Google Scholar line?

Comment (sinaatalay) [reply rendercv/data_models.py:912]: Yes, please.

File: rendercv/data_models.py (hunk 3 above)
Comment (sinaatalay) [reply rendercv/data_models.py:1586]: Yes, you can replace with that.

File: rendercv/data_models.py (hunk 4 above)
Comment (sinaatalay) [reply rendercv/data_models.py:1587]: We should delete `# project_link......` line and put the date as a GitHub link.

File: rendercv/data_models.py (hunk 5 above)
Comment (sinaatalay) [reply rendercv/data_models.py:1601]: You shouldn't uncomment it but delete it, we don't have a `project_link` field. Yes, you can update the date with the GItHub link.

File: rendercv/data_models.py (hunk 6 above)
Comment (sinaatalay) [reply rendercv/data_models.py:1614]: Let's keep the date to show users that only years are acceptable too. I agree, it seems irrelevant.

File: rendercv/data_models.py (hunk 7 above)
Comment (sinaatalay) [reply rendercv/data_models.py:1613]: We can put the GitHub link.

File: rendercv/data_models.py (hunk 1 above)
Comment (sinaatalay) [reply rendercv/data_models.py:912]: I will take care of the tests, and we shouldn't include Google Scholar in this pull request.

File: tests/test_data_models.py (hunk 2 above)
Comment (sinaatalay) [reply tests/test_data_models.py:430]: You shouldn't have removed that but changed `Orcid` with `ORCID`. Now, `ORCID` social network is not being tested.

Comment (TheColorDarkBlue) [reply tests/test_data_models.py:430]: I created this issue #90

File: rendercv/data_models.py (hunk 1 above)
Comment (TheColorDarkBlue) [reply rendercv/data_models.py:912]: I created this issue #85

**Review by sinaatalay — APPROVED** [review]

**Review by sinaatalay — APPROVED** [review]

**Review by sinaatalay — APPROVED** [review]

**Comment by sinaatalay** [comment]
Thank you for the pull request! It's merged.

_Note: I deleted the comments by mistake._

**Pull request merged**